### PR TITLE
added clay's data file

### PR DESCRIPTION
**Pull request by clenclen**
introduction and emoji

### CODE REVIEW

File: _data/spring_2016/clenclen.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"introduction": "hey i'm clay. i like to sleep.", 
+	"emoji": "kissing_smiling_eyes"
```
Comment (afeld) [inline _data/spring_2016/clenclen.json:3]: Great choice! :+1:

**Comment by afeld** [comment]
Looks great, thanks for participating!

File: _data/spring_2016/clenclen.json (hunk 1 above)
Comment (bluemittens505) [inline _data/spring_2016/clenclen.json:3]: Great Job!!!!!!!!!!!

**Comment by iamakimmer** [comment]
hooray!

**Pull request merged**